examples/animation/random_walk.py

```diff
@@ -17,8 +17,12 @@ def gen_rand_line(length, dims=2):
     """
     Create a line using a random walk algorithm
 
-    length is the number of points for the line.
-    dims is the number of dimensions the line has.
+    Parameters
+    ----------
+    length : int
+        The number of points of the line.
+    dims : int
+        The dimensionality of the line.
```

Comment:
Is "dimensionality" different than "dimensions"?

Reply:
I restored "number of dimensions" to avoid this debate...